packages/admin-ui/src/page/page-content.ts

```diff
@@ -1,54 +0,0 @@
-import { Children } from 'react'
-import type { ComponentPropsWithRef } from 'react'
-import { createComponent, useElement } from '@vtex/admin-ui-react'
-import type { VariantProps } from '@vtex/admin-ui-core'
-
-import * as style from './page.style'
-
-/**
- * Page content component
- *
- * @example
- * import { Page, PageContent, PageHeader, PageHeaderTitle, PageHeaderTop } from "@vtex/admin-ui"
- *
- * <Page>
- *  <PageHeader>
- *    <PageHeaderTop>
- *      <PageHeaderTitle>
- *        Product #123
- *      </PageHeaderTitle>
- *    </PageHeaderTop>
- *  </PageHeader>
- *  <PageContent>
- *    Content
- *  </PageContent>
- * </Page>
- */
-export const PageContent = createComponent<'div', PageContentOptions>(
-  (props) => {
-    const { layout = 'standard', template, ...htmlProps } = props
-
-    const gridTemplateColumns =
-      template ?? `repeat(${Children.count(htmlProps.children)}, 1fr)`
```

Comment:
I've removed this behavior since it wasn't documented and it was generating a weird bug when the table has tabs, as you can see here:
![Captura de Tela 2023-04-03 às 17 07 04](https://user-images.githubusercontent.com/20579226/229615710-31b764ce-26ba-4b41-a22a-9c6f40e93855.png)